.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Editor specific files should not be added to git.

You can add the line `vscode` to the `.gitignore` file to exclude the directory.